server/e2e_test.go

```diff
@@ -0,0 +1,293 @@
+package server
+
+import (
+	"fmt"
+	"github.com/stretchr/testify/assert"
+	"net"
+	"strconv"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/cashshuffle/cashshuffle/message"
+)
+
+var (
+	testTempKey int
+)
+
+func TestHappyShuffle(t *testing.T) {
```

Comment:
NIT: Make sure exported methods are commented. =)

Reply:
Yes! I had not cleaned up at all. I went ahead and did some house cleaning before moving on. Also in b17bd34